BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.kt

```diff
@@ -112,14 +110,13 @@ internal class AnalyticsClient @VisibleForTesting constructor(
                 val analyticsEventDao = analyticsDatabase.analyticsEventDao()
                 val events = analyticsEventDao.getAllEvents()
                 if (events.isNotEmpty()) {
-                    val metadata = deviceInspector.getDeviceMetadata(context, sessionId, integration)
+                    val metadata = deviceInspector.getDeviceMetadata(context, configuration, sessionId, integration)
                     val analyticsRequest = serializeEvents(authorization, events, metadata)
-                    configuration?.analyticsUrl?.let { analyticsUrl ->
-                        httpClient.post(
-                            analyticsUrl, analyticsRequest.toString(), configuration, authorization
-                        )
-                        analyticsEventDao.deleteEvents(events)
-                    }
+
+                    httpClient.post(
+                            FPTI_ANALYTICS_URL, analyticsRequest.toString(), configuration, authorization
```

Comment:
I think the formatting might be off here. It looks like we should have a single tab for this new line.